src/components/FilterBar/FilterDisplay/Inputs/DateInput.react.js

```diff
@@ -25,12 +25,16 @@ export class DateInput extends TextInputBase {
 
   componentDidMount() {
     var datePickerFrom = $(React.findDOMNode(this.refs.dateRangeFrom));
-    datePickerFrom.datetimepicker({ locale: 'en-au', format: 'L' });
-    datePickerFrom.datetimepicker().on("dp.change", this.onChange.bind(this));
+    if (datePickerFrom.datetimepicker !== undefined) {
```

Comment:
this was added mainly so as to not throw errors with the example application testing